lib/index.js

```diff
@@ -0,0 +1,3 @@
+exports.DOMImplementation = require('./dom').DOMImplementation
+exports.XMLSerializer = require('./dom').XMLSerializer
```

Comment:
Question: does it make sense/ a difference to have a single vs multiple `require('./dom')` in a single file?
```suggestion
var dom = require('./dom')
exports.DOMImplementation = dom.DOMImplementation
exports.XMLSerializer = dom.XMLSerializer
```